cmd/fuzzbinder/main.go

```diff
@@ -50,8 +52,17 @@ func main() {
 		log.Fatalf("could not get absolute path for storing the fuzzing claim: %v", err)
 	}
 
+	// Get context
```

Comment:
Which context are we getting and why? Otherwise I would drop the comment.

Reply:
I agree, the comment does not add much. Also could you inline `context.Background()`?

Reply:
I removed it